sensor_msgs/JointState js
string n
float32 moving_sec
float32 dwell_sec
uint8 interpolation
uint8 INTERP_NONE=0
uint8 INTERP_LINEAR=1
# todo: add splines, etc
